developer/src/kmc-keyboard/test/fixtures/basic.txt

```diff
@@ -98,6 +98,9 @@ block(sectitems)
   62 6b 73 70
```

Comment:
basic.kmx and basic.kvk should not be in the repo as they are generated files.

Reply:
Didn't mean to add them. Maybe the fixture script did something ? 

I ran generate fixtures. Should that be gitignored ?

Reply:
Ok I see they were added.

Reply:
I think in general we don't need to generate the fixtures; we only use that for analysis of the files when the format changes. But for sure we don't add them. We could .gitignore them but I kinda like the reminder to delete them!